fallbear-cmd

```diff
@@ -2,63 +2,85 @@
 
 echo_debug() {
     if [ -n "$PVDEBUG" ]; then
-	echo $@
+        echo "$@" >&2
```

Comment:
we usually use tabs not whitespaces to indent in shell and C

Reply:
Will fix it. The original was mixed, so I didn't know.